app/views/layouts/application.html.erb

```diff
@@ -67,18 +66,33 @@
                 <li class="nav-dropdown-item"><a href="/forms/new">Forms</a></li>
               </ul>
             </li>                
-            <li>
-              <a href="#contact" class="cdc-navbar-item"><i class="fa fa-bell item-navbar-icon" aria-hidden="true"></i>
+            <li class="dropdown">
+              <a href="#notifications" class="dropdown-toggle cdc-navbar-item" id="notifications-dropdown" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><i class="fa fa-bell item-navbar-icon" aria-hidden="true"></i>
               Alerts
-              <span class="alerts-badge">1</span></a>
-            </ul>
+              <% if current_user.notifications.exists? and current_user.notifications.where('read' => false).count > 0 %>
```

Comment:
Should I make a separate react widget for the alerts menu item, the react component introduced in this PR is the Notification menu displayed in the drop down and the notification items in that list. The actual nav bar isn't a widget yet so I guess I would assume the logic for the badge itself would be in that component, but I could make a temporary component just for the Notifications cdc-navbar-item, or I could reactify the nav bar in a separate PR.